### PR TITLE
Fix incorrect 1123label validation

### PR DESCRIPTION
**Pull request by hanxiaop**
**Please provide a description of this PR:**

### CODE REVIEW

File: pkg/config/labels/instance.go [hunk 1]
```diff
@@ -25,7 +25,7 @@ import (
 
 const (
 	DNS1123LabelMaxLength = 63 // Public for testing only.
-	dns1123LabelFmt       = "[a-zA-Z0-9](?:[-a-zA-Z0-9]*[a-zA-Z0-9])?"
+	dns1123LabelFmt       = "[a-z0-9](?:[-a-z0-9]*[a-z0-9])?"
```
Comment (hzxuzhonghu) [inline pkg/config/labels/instance.go:28]: Unlike the k8s, we are using this to validate Kind, DNS 1123 can contain upper cases

https://github.com/kubernetes/kubernetes/pull/39675/files

**Review by howardjohn — CHANGES_REQUESTED** [review]
This seems like a breaking change. Is there any more context on why we should do this?

File: pkg/config/labels/instance.go (hunk 1 above)
Comment (hanxiaop) [reply pkg/config/labels/instance.go:28]: @hzxuzhonghu Thanks

**Comment by hanxiaop** [comment]
> This seems like a breaking change. Is there any more context on why we should do this?

@howardjohn I was trying to add some dns1123label validation to iop fields in validate, and I think the uppercase should be checked as well. I'll close this one and do it in another way.

**Pull request closed without merging**